### PR TITLE
Feature/multiple image

### PR DESCRIPTION
**Pull request by akopp90**

### CODE REVIEW

File: components/layout/ActivityForm.js [hunk 1]
```diff
@@ -136,6 +155,24 @@ export default function ActivityForm({
     setCategories([]);
   }
 
+  function handleDeleteImage(imageToDelete) {
```
Comment (F-Kirchhoff) [inline components/layout/ActivityForm.js:158]: please use `imageUrl` here instead for clarity.

File: components/ui/Button.js [hunk 2]
```diff
@@ -25,6 +27,10 @@ const StyledButton = styled.button`
   display: flex;
   font: inherit;
   cursor: pointer;
+  position: ${(props) => (props.$deleteImage ? "relative" : "unset")};
+  z-index: ${(props) => (props.$deleteImage ? 1 : "unset")};
+  top: ${(props) => (props.$deleteImage ? 0 : "unset")};
+  right: ${(props) => (props.$deleteImage ? 0 : "unset")};
```
Comment (F-Kirchhoff) [inline components/ui/Button.js:33]:
```suggestion
${(props) => (props.$deleteImage && css`
  position: "relative"
  z-index: 1;
  top: 0;
  right: 0;
`
```

File: components/ui/Upload.js [hunk 3]
```diff
@@ -1,18 +1,39 @@
...
+
+function Upload({ name, isRequired, children, onChange, multiple }) {
+  const inputRef = useRef(null);
+
+  const handleDragOver = (event) => {
+    event.preventDefault();
+  };
+
+  const handleDrop = (event) => {
+    e.preventDefault();
```
Comment (F-Kirchhoff) [inline components/ui/Upload.js:12]:
```suggestion
    event.preventDefault();
```

File: components/ui/Upload.js [hunk 4]
```diff
@@ -1,18 +1,39 @@
...
+    const files = evenzt.dataTransfer.files;
```
Comment (F-Kirchhoff) [inline components/ui/Upload.js:13]:
```suggestion
    const files = event.dataTransfer.files;
```

**Review by Bianca-Gundel — APPROVED** [review]

**Pull request merged**